src/components/app/MenuButtons/MetaInfo.js

```diff
@@ -150,7 +150,7 @@ class MetaInfoPanelImpl extends React.PureComponent<Props> {
             <>
               <div className="metaInfoRow">
                 <span className="metaInfoLabel">Buffer Capacity:</span>
-                {formatBytes(configuration.capacity)}
+                {formatBytes(configuration.capacity * 8, 0)}
```

Comment:
nit: It would be also good to add a comment here explaining why we are multiplying it with 8 (like in one of the new comments in test).